README.md

```diff
@@ -1 +1,178 @@
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 To Do List <a name="about-project"></a>
+
+This is a list of tasks that an individual needs to complete or accomplish. Tasks are typically put in order by priority or importance. 
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>html</li>
+    <li>css</li>
+    <li>javascript</li>
+    <li>webpack</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- *Add what to do*
+- *Remove after completed*
+- *Great UI*
+- 
+<!-- Live Demo -->
+
+### Live Demo <a name="live-demo"></a>
+
+[visit here](https://oluwatobi-fasasi.github.io/toDoList/dist/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need the following tools:
+- Node.js in your computer
+- webpack
+- IDE(e.g.: Vscode,...)
+- Terminal
+ 
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+Example commands:
+
+```sh
+  git clone https://github.com/oluwatobi-fasasi/To-Do-List
+```
+
+
+### Install
+
+- Install this project with:
+
+```sh
+  cd To-Do-List
+
+  npm install or npm i
+```
+
+- Run an optimized build for prodction:
+```
+  npm run build
+```
+
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm run dev
+
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  npm run test
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@oluwatobi](https://github.com/oluwatobi-fasasi)
+- Twitter: [@Fasasi](https://twitter.com/FasasiTobi1)
+- LinkedIn: [Fasasi](https://www.linkedin.com/in/fasasi-abdulwasih-oluwatobi-129a8b109/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] *add responsiveness*
+- [ ] *add more UX*
+
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Follow me on my github if you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse as well as my code partner(s)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](LICENSE.md) licensed.
+
```

Comment:
![image](https://github.com/oluwatobi-fasasi/toDoList/assets/93039359/d4fc9f0e-ef6d-41a2-b8c3-135f15eeb06b)

- [ ] It would be nice if you fix your MIT link because it is broken and giving me a 404 error which you can see in the image above 👀 Kindly fix this error to make your readme descriptive and professional 👍 This error is coming because you don't have a LICENSE file in your repo 🤔 Kindly add a LICENSE file using these [guidelines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to make your project licensed and readme descriptive 😊